docs/extended_api/synchronization_library/pipeline/quit.md

```diff
@@ -0,0 +1,19 @@
+---
+nav_exclude: true
+---
+
+# cuda::pipeline\<Scope>::**quit**
+
+```c++
+bool quit();
+```
+
+Quits the current thread's participation in the collective ownership of the corresponding _shared state_ ([`cuda::pipeline_shared_state`](../pipeline_shared_state.md)). Ownership of the _shared state_ is released by the last invoking thread.
+
+## Return value
+
+`true` if ownership of the _shared state_ was released, otherwise `false`.
+
+## Notes
+
+The behavior undefined if any operation other than [`~pipeline`](./destructor.md) is issued by the current thread after quitting.

```

Comment:
Can we also add this note as a precondition to all the functions that are no longer valid to call after `quit`?

Reply:
I avoided doing that because it would extend to operations on the pipeline (e.g. `memcpy_async`) and I judged it too verbose. Are you thinking about only doing it for `pipeline` member functions?

Reply:
We should do it everywhere where we take a pipeline and have a precondition on the pipeline. We should avoid burrying it in notes of just a single member function; that's easy to overlook.

Reply:
Done.